<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>516</width>
    <height>461</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(50, 50, 50);


</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>24</y>
      <width>411</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei</family>
      <pointsize>18</pointsize>
      <weight>3</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
background-color: rgb(100, 100, 100);
color: rgb(35, 35, 35);
border-radius: 6px;
font: 25 18pt &quot;Microsoft YaHei&quot;;
}

QLineEdit:focus{
color: rgb(35, 35, 35);
border-radius: 6px;


}


</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>82</y>
      <width>518</width>
      <height>331</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei</family>
      <pointsize>18</pointsize>
      <weight>3</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(50, 50, 50);
color: rgb(158, 158, 158);
font: 25 18pt &quot;Microsoft YaHei&quot;;

</string>
    </property>
    <item>
     <property name="text">
      <string>New Item</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>New Item</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>New Item</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>New Item</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>55</x>
      <y>422</y>
      <width>410</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: rgb(100, 100, 100);
color: rgb(35, 35, 35);
border-radius: 6px;
}

QPushButton:hover{
background-color: rgb(130, 130, 130);
color: rgb(35, 35, 35);
border-radius: 6px;
}



</string>
    </property>
    <property name="text">
     <string>Добавить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>433</x>
      <y>24</y>
      <width>75</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: rgb(100, 100, 100);
color: rgb(35, 35, 35);
border-radius: 6px;
}

QPushButton:hover{
background-color: rgb(130, 130, 130);
color: rgb(35, 35, 35);
border-radius: 6px;
}



</string>
    </property>
    <property name="text">
     <string>Поиск</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
